WordPress/Classes/ViewRelated/Stats/Charts/StatsLineChartView.swift

```diff
@@ -246,22 +254,39 @@ private extension StatsLineChartView {
         // This adjustment is intended to prevent clipping observed with some labels
         // Potentially relevant : https://github.com/danielgindi/Charts/issues/992
         extraTopOffset = Constants.topOffset
-    }
-
-    func configureYAxisMaximum() {
-        let lowestMaxValue = Double(Constants.verticalAxisLabelCount - 1)
 
-        if let maxY = data?.getYMax(axis: .left),
-           maxY >= lowestMaxValue {
-            leftAxis.axisMaximum = VerticalAxisFormatter.roundUpAxisMaximum(maxY)
+        guard let data else { return }
+        if areDataValuesIdentical {
+            yAxis.axisMinimum = 0
+            yAxis.axisMaximum = data.getYMax(axis: .left) * 2
         } else {
-            leftAxis.axisMaximum = lowestMaxValue
+            if statType == .viewsAndVisitors {
+                yAxis.axisMinimum = 0
+
+                let lowestMaxValue = Double(Constants.verticalAxisLabelCount - 1)
+                let dataYMax = data.getYMax(axis: .left)
+                if dataYMax >= lowestMaxValue {
+                    yAxis.axisMaximum = VerticalAxisFormatter.roundUpAxisMaximum(dataYMax)
+                } else {
+                    leftAxis.axisMaximum = lowestMaxValue
+                }
+            } else if statType == .subscribers {
+                yAxis.axisMinimum = data.getYMin(axis: .left)
+                yAxis.axisMaximum = data.getYMax(axis: .left)
```

Comment:
I didn't use `VerticalAxisFormatter.roundUpAxisMaximum()` here because it results in y-axis maximums that are too big. However, the downside to not using it is that the y-axis values aren't "neat" round numbers e.g. "1.57k" might be used instead of "1.8k" as one of the y-axis tick labels.